hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/dsl/operations/transactions/AssociateTokensOperation.java

```diff
@@ -32,7 +35,12 @@
  */
 public class AssociateTokensOperation extends AbstractSpecTransaction<AssociateTokensOperation, HapiTokenAssociate>
         implements SpecOperation {
+    @Nullable
     private final SpecAccount account;
+
+    @Nullable
+    private final SpecContract contract;
```

Comment:
Is the idea that we will be adding operations like this in the future to keep the tests as clean as possible?  For example, I need an `approveAllowance` operation for the current set of tests that I am writing.  Hopefully the set of operations will remain somewhat manageable.  I really like the way the ultimate test turned out.